pybliometrics/scopus/utils/create_config.py

```diff
@@ -2,7 +2,8 @@
 from typing import List, Optional
 
 
-def create_config(keys: Optional[List[str]] = None,
+def create_config(config_dir: str,
```

Comment:
`config_dir` should also be optional. When `None`, use the default from `CONFIG_FILE`.